examples/guestbook/README.md

```diff
@@ -426,39 +489,56 @@ spec:
         - containerPort: 80
 ```
 
-[Download example](frontend-controller.yaml?raw=true)
-<!-- END MUNGE: EXAMPLE frontend-controller.yaml -->
+[Download example](frontend.yaml?raw=true)
+<!-- END MUNGE: EXAMPLE frontend.yaml -->
 
-Using this file, you can turn up your frontend with:
+#### Using 'type: LoadBalancer' for the frontend service (cloud-provider-specific)
+
+For supported cloud providers, such as Google Compute Engine or Google Container Engine, you can specify to use an external load balancer
+in the service `spec`, to expose the service onto an external load balancer IP.
+To do this, uncomment the `type: LoadBalancer` line in the `frontend-service.yaml` file before you start the service.
+
+[See the section below](#accessing-the-guestbook-site-externally) on accessing the guestbook site externally for more details.
```

Comment:
Either move "Accessing the guestbook site externally" below this section or change it to "**Appendix:** Accessing the guestbook site externally" and change "See the section below" to something like "See the **appendix** below".